manualdirectdebit.php

```diff
@@ -226,6 +233,7 @@ function manualdirectdebit_civicrm_pageRun(&$page) {
     $pageProcessor = new CRM_ManualDirectDebit_Hook_PageRun_TabPage();
     $pageProcessor->setContributionId($contributionId);
     $pageProcessor->hideDirectDebitFields();
+    $pageProcessor->changeRecurringContributionButtons();
```

Comment:
There's a hook you can use to add/alter the list of actions of each row, basically for everything. This is an example of how we implemented the hook on membershipextras extension, to do something similar, also for recurring contributions:

```php
function membershipextras_civicrm_links($op, $objectName, $objectId, &$links, &$mask, &$values) {
  if ($op == 'contribution.selector.recurring' && $objectName == 'Contribution') {
    $recurContribuLinksHook = new CRM_MembershipExtras_Hook_Links_RecurringContribution($links);
    $recurContribuLinksHook->alterLinks();
  }
}
```

Please implement this hook to add the action to the list of links for each row.

The hook is documented here:
https://docs.civicrm.org/dev/en/latest/hooks/hook_civicrm_links/

Thanks!